patches/dmd/0001-Use-sc-transitional.ini-dmd-transitional.conf.patch

```diff
@@ -1,18 +1,18 @@
-From 0c6d41aa940026cbe15a736f69a80805114079bc Mon Sep 17 00:00:00 2001
+From 3119977c841d128c63496b931e1f7f4f20a49a37 Mon Sep 17 00:00:00 2001
 From: Mihails Strasuns <mihails.strasuns.contractor@sociomantic.com>
 Date: Tue, 9 Feb 2016 20:37:37 +0200
-Subject: [PATCH 01/21] Use sc-transitional.ini/dmd-transitional.conf
```

Comment:
So, I guess we need less patches because things we are backporting are already in v2.071.2, right?